site/en/docs/crux/api/index.md

```diff
@@ -0,0 +1,592 @@
+---
+# Required
+layout: 'layouts/doc-post.njk'
+
+# Required
+title: CrUX API
+
+# Required
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in
+# Google Search.
+description: >
+  Learn how to construct requests to and parse responses from the CrUX API
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  The Chrome UX Report API gives low-latency access to aggregated Real User Metrics (RUM) from the Chrome User Experience Report.
+
+# Required
+date: 2022-06-23
+
+# Optional
+# Include an updated date when you update your post
+# updated: 2020-10-16
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - simonhearne
+
+# Optional
+# How to a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - performance
+---
+
+The Chrome UX Report API gives low-latency access to aggregated Real User Metrics (RUM) from the Chrome User Experience Report.
+
+## Common use case
+
+The Chrome UX Report API allows for the querying of user experience metrics for a specific URI like "Get metrics for the `https://example.com` origin."
+
+## Data model
+
+### Record
+
+A discrete piece of information about a page, or site. A record can have data that is specific for an identifier and for a specific combination of dimensions. A record can contain data for one or more metrics.
+
+### Identifiers
+
+Identifiers specify what records should be looked up. In the Chrome UX Report these identifiers are webpages and websites.
+
+### Origin
+
+When the identifier is an origin all data present for all pages in that origin are aggregated together. For example, say the http://www.example.com origin had pages as laid out by this sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+This would mean that when querying the Chrome UX Report with the origin set to `http://www.example.com`, data for `http://www.example.com/`, `http://www.example.com/foo.html`, and `http://www.example.com/bar.html` would be returned because those are all pages under that origin.
```

Comment:
```suggestion
This would mean that when querying the Chrome UX Report with the origin set to `http://www.example.com`, data for `http://www.example.com/`, `http://www.example.com/foo.html`, and `http://www.example.com/bar.html` would be returned, aggregated together, because those are all pages under that origin.
```